data/core/macros/abilities.cfg

```diff
@@ -387,3 +387,23 @@ Enemy units cannot see this unit while it is in deep water, except if they have
         special_note={INTERNAL:SPECIAL_NOTES_FEEDING}
     [/dummy]
 #enddef
+
+#define ILLUMINATES_HALOS ID HALO
```

Comment:
Either the macro name should change, or it shouldn't take an ID.

```
#define ILLUMINATES_HALO HALO
  # Provide a halo image which is only shown when the unit's ABILITY_ILLUMINATES is active
  ...
```

```
#define ADD_ABILITY_HALO ID HALO
  # Show the image HALO above the unit when the ability with id ID is active
  ...
```

Reply:
> Either the macro name should change, or it shouldn't take an ID.
`#define ADD_ABILITY_HALO ID HALO
  # Show the image HALO above the unit when the ability with id ID is active
  ...`

second option choose

Reply:
> Either the macro name should change, or it shouldn't take an ID.
> 
> ```
> 
> ```
> #define ADD_ABILITY_HALO ID HALO
>   # Show the image HALO above the unit when the ability with id ID is active
>   ...
> ```

second choice